index.html

```diff
@@ -902,32 +902,64 @@
 These threats combine into the particular concrete threats we want web
 specifications to defend against, described in subsections here:
 
-## Unwanted same-site recognition {#hl-recognition-same-site}
+## Unwanted cross-context recognition {#hl-recognition-cross-context}
 
 Contributes to [=surveillance=], [=correlation=], and [=identification=].
 
-Users of most instantiations of the web platform expect that if they visit a
-site on one day, and then visit again the next day, the site will be able to
-recognize that they're the same user. This allows sites to save the user's
-preferences, shopping carts, etc. The web platform offers many mechanisms that
-are either intended to accomplish this recognition or that can be trivially used
-for it, including [[RFC6265|cookies]], {{WindowLocalStorage/localStorage}},
-{{WindowOrWorkerGlobalScope/indexedDB}}, {{CacheStorage}}, and other forms of
-storage.
+As described in [[[#identity]]], [=cross-context recognition=] can sometimes be [=appropriate=], but
+users need to be able to control when websites do it as much as possible.
 
-A privacy harm only occurs if the user wants to break the association between
-two visits, but the site can still determine with high probability that the two
-visits came from the same user.
+<div class="practice">
 
-A user might expect that their two visits won't be associated if they:
+<p><span class="practicelab" id="principle-prevent-cross-partition-recognition">[=User agents=]
```

Comment:
(Sorry for the ReSpec ignorance, but) can't this be simplified to `<p class="practicelab ...>`? Why the nested span?